interfaces/builtin/basedeclaration.go

```diff
@@ -32,15 +32,103 @@ type: base-declaration
 authority-id: canonical
 series: 16
 revision: 0
-plugs:
-  network: true
```

Comment:
The spec doc had a few plugs with `allow-installation: false`, I guess it is ok for now to not have it since we don't have snap declarations.